README.md

```
@@ -178,20 +178,22 @@ To quickly get hands on LCS, we can run it using the default configurations prov
    ```bash
    export OPENAI_API_KEY=sk-xxxxx
    ```
-3. start Llama stack server
+3. start LCS server 
    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)